Kitodo/src/main/java/org/kitodo/production/helper/SearchResultGeneration.java

```diff
@@ -60,40 +55,19 @@ public SXSSFWorkbook getResult() {
         return getWorkbook();
     }
 
-    private List<ProcessDTO> getResultsWithFilter() {
-        List<ProcessDTO> processDTOS = new ArrayList<>();
-        try {
-            processDTOS = ServiceManager.getProcessService().findByQuery(getQueryForFilter(ObjectType.PROCESS),
-                ServiceManager.getProcessService().sortById(SortOrder.ASC), true);
-        } catch (DataException e) {
-            logger.error(e.getMessage(), e);
+    private List<Process> getResultsWithFilter() {
+        BeanQuery query = new BeanQuery(Process.class);
+        if (StringUtils.isNotBlank(filter)) {
+            query.restrictWithUserFilterString(filter);
         }
-
-        return processDTOS;
-    }
-
-    /**
-     * Gets the query with filters.
-     *
-     * @param objectType Type of object that should be filtered
-     * @return A BoolQueryBuilder
-     */
-    public BoolQueryBuilder getQueryForFilter(ObjectType objectType) {
-        BoolQueryBuilder query = new BoolQueryBuilder();
-
-        try {
-            query = ServiceManager.getFilterService().queryBuilder(this.filter, objectType, false, false);
-        } catch (DataException e) {
-            logger.error(e.getMessage(), e);
-        }
-
         if (!this.showClosedProcesses) {
-            query.mustNot(ServiceManager.getProcessService().getQueryForClosedProcesses());
+            query.restrictToNotCompletedProcesses();
         }
         if (!this.showInactiveProjects) {
-            query.mustNot(ServiceManager.getProcessService().getQueryProjectActive(false));
+            query.addBooleanRestriction("project.active", Boolean.FALSE);
         }
-        return query;
+        query.performIndexSearches();
```

Comment:
```suggestion
        if (!this.showInactiveProjects) {
            query.addBooleanRestriction("project.active", Boolean.TRUE);
        }
        query.performIndexSearches();

```
You are defaulting to Boolean.FALSE, which breaks Excel file generation because only processes of inactive projects are included.